src/main/kotlin/com/exactpro/th2/common/ConfigurationProviderFactory.kt

```diff
@@ -29,18 +31,15 @@ interface ConfigurationProviderFactory {
     val type: Class<out ConfigurationProvider>
 
     /**
-     * Method for instantiating a [ConfigurationProvider] using parameters.
-     * Each factory implementation can have its own arguments
-     *
-     * To get an array of arguments, use method [parseArguments]
+     * Method for creating empty object, that will be filled with
+     * required settings for specific implementation.
      */
-    fun create(args: Array<String>): ConfigurationProvider
+    fun settings(): ConfigurationProviderConfig
```

Comment:
Looks like redundant, because we can do that using class and default constructor
val settingsType: Class<out ConfigurationProviderConfig> look be better